### PR TITLE
Feature/align pairwise

### PR DESCRIPTION
**Pull request by rrahn**
Adds align_pairwise function.
I will also add documentation later this evening.

### CODE REVIEW

File: include/seqan3/alignment/pairwise/align_pairwise.hpp [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+#include <seqan3/std/concepts>
+#include <seqan3/std/ranges>
+
+namespace seqan3
+{
+
+//!\cond
+template <std::ranges::InputRange sequence_t, typename alignment_config_t>
+    requires detail::is_algorithm_configuration_v<remove_cvref_t<alignment_config_t>> &&
+             tuple_like_concept<decltype(*ranges::begin(std::declval<sequence_t &>()))>
```
[the remark this answers is not in the document]
Comment (rrahn) [reply include/seqan3/alignment/pairwise/align_pairwise.hpp]: that is not the bug, and what do you suggest? that your code is more readable? ts ts ts

**Comment by rrahn** [comment]
I found the bug already

**Comment by marehr** [comment]
:+1: :+1:

File: include/seqan3/alignment/pairwise/align_pairwise.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (marehr) [reply include/seqan3/alignment/pairwise/align_pairwise.hpp]: yeah :P

**Comment by marehr** [comment]
note: that the score changed to a negative output

File: include/seqan3/alignment/pairwise/align_pairwise.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (h-2) [reply include/seqan3/alignment/pairwise/align_pairwise.hpp]: Just do `reference_t<seq_t>` please!

**Pull request merged**